kubernetes/zenko/charts/cloudserver/templates/_env.tpl

```diff
@@ -20,6 +20,22 @@ env:
     value: "{{ .Release.Name }}-cloudserver,{{ .Values.endpoint }}"
   - name: HEALTHCHECKS_ALLOWFROM
     value: "{{ .Values.allowHealthchecksFrom }}"
+  - name: AWS_S3_HTTPAGENT_KEEPALIVE
+    value: "{{ .Values.externalBackends.aws_s3.keepAlive }}
+  - name: AWS_S3_HTTPAGENT_KEEPALIVE_MS
+    value: ""{{ .Values.externalBackends.aws_s3.keepAliveMsecs }}
+  - nmae: AWS_S3_HTTPAGENT_KEEPALIVE_MAX_SOCKETS
+    value: "{{ .Values.externalBackends.aws_s3.maxSockets }}"
+  - name: AWS_S3_HTTPAGENT_KEEPALIVE_MAX_SOCKETS
+    value: "{{ .Values.externalBackends.aws_s3.maxFreeSockets }}"
+  - name: GCP_HTTPAGENT_KEEPALIVE
+    value: "{{ .Values.externalBackends.gcp.keepAlive }}
+  - name: GCP_HTTPAGENT_KEEPALIVE_MS
+    value: ""{{ .Values.externalBackends.gcp.keepAliveMsecs }}
+  - nmae: GCP_HTTPAGENT_KEEPALIVE_MAX_SOCKETS
+    value: "{{ .Values.externalBackends.gcp.maxSockets }}"
+  - name: GCP_HTTPAGENT_KEEPALIVE_MAX_SOCKETS
```

Comment:
Looks like the name in the cloudserver commit is different: https://github.com/scality/cloudserver/pull/1611/files#diff-60713fb19f6c50a2473835df63ff17f9R185